tdclient/api.py

```diff
@@ -638,13 +634,13 @@ def value(s):
                 return s
 
         if columns is None:
-            reader = csv.DictReader(getreader(file_like), dialect=dialect)
+            reader = csv.DictReader(io.TextIOWrapper(file_like, encoding), dialect=dialect)
```

Comment:
It seems there are similar code in test's helper methods. It'd be better changing these codes as well in this time.

https://github.com/treasure-data/td-client-python/blob/770e2e104631b27c05d075ecabc9d2a180c71e15/tdclient/test/test_helper.py#L114

Reply:
Removed as well 1c6f93d 120f791